README.md

```diff
@@ -1,43 +1,153 @@
 
-Add and remove exercises
-Add and remove sets
-For each set, add weight and reps
-For each set, mark as finished or unfinished
+### Wireframe using Figma 
+  ### Getting Started The App build out with simple wireframes using Figma.
+<img src="/assets/images/homewireframe.png"
+        alt="home page"
+             height="80">
+<img src="/assets/images/signinwireframe.png"
+        alt="sign in page"
+             height="80">
+<img src="/assets/images/signupwireframe.png"
+        alt="sign up page"
+             height="80">
+### How To Backup & Restore Your Workouts
+You can export individual workouts or create an export of all your workouts and import them later. You can also transfer them to another device.
 
+<!-- GETTING STARTED -->
+## Getting Started
 
-if first time signin
-take to profile
-auto profile with info
+This is an example of how you may give instructions on setting up your project locally.
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+* npm
+  ```sh
+  npm install npm@latest -g
+  ```
+
+### Installation
+### Installation
+_Below is an example of how you can instruct your audience on installing and setting up your app. This template doesn't rely on any external dependencies or services._
+### Installation
+1. Get a free account at for firebase to create your own collection and add it to your .env file to connect your API to your Data base
+2. Clone the repo
+   ```sh
+   git clone https://github.com/sisay2405/phiscal-fitness-app
+   ```
+3. Install NPM packages
+   ```sh
+   npm install
+   ```
+   ### Installation
+4. Enter your firebase link in your `.env` file in your backend 
+   ```js
+   REACT_APP_FIREBASE_API_KEY = 'ENTER YOUR API KEY';
+   REACT_APP_FIREBASE_AUTH_DOMAIN = 'ENTER YOUR FIREBASE_AUTH_DOMAIN';
+   REACT_APP_FIREBASE_PROJECT_ID = 'ENTER YOUR FIREBASE_PROJECT_ID';
+   REACT_APP_FIREBASE_STORAGE_BUCKET = 'ENTER YOUR FIREBASE_STORAGE_BUCKET';
+   REACT_APP_FIREBASE_MESSAGING_SENDER_ID = 'ENTER YOUR FIREBASE_MESSAGING_SENDER_ID';
+   REACT_APP_FIREBASE_APP_ID = 'ENTER YOUR FIREBASE_APP_ID';
+   ```
+5. npm start for START the app.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- USAGE EXAMPLES -->
+## Usage
+
+Can be used to workout the Exercises your can create your own phisical finteness web app and share it with your community. You can:
+
+- [x] creat, customezie Exercises
+- [x] Add Exercises
+- [x] Delete Exercises
+- [x] Edit Exercises
+- [x] User Sign In/Sign Up
+    - [x] To creat your Exercises you will need to sign in/sign up
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- CONTACT -->
+## Contact
+
+Sisay Areaya - sareaya@alphaworks.tech , abrehasisay04@gmail.com
+
+Project Link: [https://github.com/sisay2405/phiscal-fitness-app]https://github.com/sisay2405/phiscal-fitness-app
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
These 'back to top' links don't seem to work correctly.

Reply:
The 'back to top' links now work properly, but aren't always showing up on a new line and are sometimes showing up on the same line as existing text.